test/client_test.js

```diff
@@ -28,6 +28,7 @@ describe('Client', () => {
     beforeEach(function () {
       nock(baseUrl)
         .matchHeader('accept', 'application/json+fhir')
+        .matchHeader('Authorization', '')
         .get('/metadata')
```

Comment:
This is appropriate, here, because requests for `/metadata` should work without authorization?

Reply:
`/metadata` is where you get the necessary OAuth urls, so it needs to be accessible without authorization.

Reply:
We need to remove this before merging.

Reply:
This is testing that the 'Authorization' header is empty, when requesting `/metadata`

Reply:
This is testing that the 'Authorization' header is *empty*, when requesting `/metadata`